### ISSUE TITLE
Add product, model and geometry to ALT+Click menu inside CET

### ISSUE THREAD

**Issue by tucaz** [Allsteel/awesome#203]
From Jeff @ Configura

`if you override viewSources on your snapper you can call a function called invokeViewSource(object, #Function, show)`

This way we can add model, products and geometry to the ALT+Click function inside CET

**Comment by twlamb** [comment]
Example implementation from `PanelFrame` in cm.abstract.office:
```
public void viewSources(bool show=true) {
    super(show);
    if (_layout) invokeViewSource(_layout, #Function, show);
    if (officeFrameProxy) invokeViewSource(officeFrameProxy, #Function, show);
}
```

**Comment by twlamb** [comment]
Fixed in Allsteel/acs-cet-extension#976
